pkg/beacon/relay/node.go

```diff
@@ -122,23 +137,39 @@ func (n *Node) JoinGroupIfEligible(
 					return
 				}
 
-				// final broadcast channel name for group is the compressed
-				// public key of the group
+				// The final broadcast channel name for group is the compressed
+				// public key of the group.
 				channelName := hex.EncodeToString(
```

Comment:
Can we rename it to `groupPublicKey` or something similar? Even if `groups.go` expects the channel name, we can pass `groupPublicKey` as a channel name and in the context of `node.go` we do not expect the channel but the group public key to be logged.

Reply:
See 87e9b770

Reply:
See 5ac87dfa